PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
+        [String]$command,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Name of the command')]
+        [String]$name,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Command Type')]
+        [string]$commandType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Launch Type')]
+        [string]$launchType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Listens To')]
+        [string]$listensTo,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Schedule')]
+        [string]$schedule,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The name of the command trigger')]
+        [string]$trigger,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'When the command will repeat')]
```

Comment:
"The scheduled command repeat type of the JumpCloud Command you wish to search for."

And validate set:
[ValidateSet('minute', 'hour', 'day', 'week', 'month')]